src/io.ts

```diff
@@ -89,7 +89,7 @@ export function outputResult(
     );
     fs.outputFileSync(fullCodeOutPath, code);
 
-    if (options.formatCairo || options.dev) {
+    if (options.formatCairo /* || options.dev : Cairo-format is disabled, as it has a bug */) {
```

Comment:
Do you mean remove the flag completely? I think its okay to suspend its actions for now because Cairo 1.0 compiler will have its own flag and in the upgrade we can enable it again

Reply:
yes fixed